umh-core/pkg/service/redpanda/redpanda.go

```
@@ -0,0 +1,853 @@
+	IsMetricsErrorFree(metrics Metrics) bool
+	// HasProcessingActivity checks if a Redpanda service has processing activity
+	HasProcessingActivity(status RedpandaStatus) bool
+}
+
+// HTTPClient interface for making HTTP requests
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// defaultHTTPClient is the default implementation of HTTPClient
+type defaultHTTPClient struct {
```

golangci-lint (1.64.8)

type `defaultHTTPClient` is unused

(unused)